src/app/Traits/DbUniqueIdentifier.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace NetworkRailBusinessSystems\UserLogin\Traits;
+
+trait DbUniqueIdentifier
```

Comment:
Not a great name for a trait. If you look at the traits we commonly use, they are associative phrases such as `HasPermissions` or `UsesFactories`. Consider something along the lines of `HasGuidInDatabase`.

Reply:
renamed it